src/Graphics/Vty/Image/Internal.hs

```diff
@@ -82,7 +82,7 @@ data Image =
     | HorizJoin
       { partLeft :: Image 
       , partRight :: Image
-      , outputWidth :: Int -- ^ imageWidth partLeft == imageWidth partRight. Always > 1
+      , outputWidth :: Int -- ^ imageWidth partLeft == imageWidth partRight. Always > 0
```

Comment:
What you're saying sounds right to me, in which case the comment is wrong. Although it does beg the question: what did its author intend?